lib/ClimaLandSimulations/src/fluxnet_simulation.jl

```diff
@@ -34,11 +35,15 @@ function fluxnet_simulation(
         "utilities",
         "timestepper_setup.jl",
     ),
+    pft_pcts = "None",

```

Comment:
P.S this is just a temporary fix to be able to run the model with some PFTs right now, will clean this up later.